modules/ROOT/pages/_partials/osv2-faq.adoc

```diff
@@ -13,7 +13,7 @@ On-premises Mule applications can access Object Store v2 only via the Object Sto
 //end::faqOnPrem[]
 
 //tag::faqKeyChars[]
-Object Store v2 doesn't support using pipe (`|`) characters in keys.
+Object Store v2 doesn't support using pipe (`|`) characters in keys and converts any spaces to plus (`+`) characters.
```

Comment:
Only OSv2 plugin does this. The REST API doesn't do this conversion, it takes user input key and stores it as is in the backend.